### DISCUSSION TITLE
Controlling the output verbosity of Module.verify execution

### DISCUSSION THREAD

**Discussion by ziglee** [InsertKoinIO/koin#1629] [Ideas]
The `Module.verify` (package `org.koin.test.verify.verify` from `io.insert-koin:koin-test-junit4:3.3.0`) is very useful for testing the modules but it can generate a log of console output that in some cases is not needed.
The idea would be to allow the user to control the verbosity of this output via an argument of the function verify.

**Comment by arnaudgiuliani** [comment]
Hello,

do you mean shuting down all logs if needed?

**Reply by ziglee** [reply]
That could be one solution :)
